entries/l/logitech.com.json

```
@@ -0,0 +1,18 @@
+{
+  "Logitech": {
+    "domain": "logitech.com",
+    "additional-domains": [
+      "astrogaming.com",
+      "logi.com"
+    ],
+    "img": "logitech.com.png",
+    "tfa": [
+      "email",
+      "totp"
+    ],
+    "keywords": [
+      "other",
+      "gaming"
```

The __gaming__ category is for games and game-related services such as Steam and Origin whilst the __other__ category is for things that can't be categorized.
I suggest you use the __retail__ category instead and Logitech is primarily in the business of selling consumer products.
```suggestion
      "retail"
```

---

Sure